common/src/main/kotlin/org/valkyrienskies/tournament/TournamentBlocks.kt

```diff
@@ -56,6 +57,7 @@ object TournamentBlocks {
         BALLAST                  = register("ballast", ::BallastBlock)
         POWERED_BALLOON          = register("balloon", ::PoweredBalloonBlock)
         BALLOON                  = register("balloon_unpowered", ::BalloonBlock)
+        FLOATER                  = register("floater", ::FloaterBlock)
```

Comment:
Could you replace this line with the following code, and remove the FloaterBlock class:
```kotlin
FLOATER                  = register("floater") { Block(
    Properties.of(Material.WOOD)
        .sound(SoundType.WOOD)
        .strength(1.0f, 2.0f)
) }
```
and also change line 38 to use Block in the type argument instead of FloaterBlock